src/js/contentscript-goodblock.jsx

```diff
@@ -172,21 +188,23 @@ var registerContentScriptChangeListener = function() {
 var initGoodblock = function() {
 
 	// Make sure our base elem exists.
-	var baseElem = getOrCreateBaseElem();
+	//var baseElem = getOrCreateBaseElem();
+
+	var iFrameElem = getOrCreateBaseIFrame();
 
 	// Increment the saved number of Goodblock content scripts
 	// we've injected, and save that value in this content script.
-	CONTENT_SCRIPT_NUM = incrementGoodblockContentScriptNum();
+	//CONTENT_SCRIPT_NUM = incrementGoodblockContentScriptNum();
```

Comment:
We need this (and some of the other code you commented out) to make sure we don't run the content script multiple times in the same page (e.g. when the extension updates). I'll fix this.